modules/microshift-nw-network-policy-intro.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-network-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-nw-network-policy-intro_{context}"]
+= How network policy works in {microshift-short}
+
+In a cluster using the default OVN-Kubernetes Container Network Interface (CNI) plugin for {microshift-short}, network isolation is controlled entirely by `NetworkPolicy` objects.
+//Q: true for microshift?
```

Comment:
In order to control traffic you have firewalld (outside MicroShift) and `NetworkPolicy` (inside MicroShift). A `NetworkPolicy` works only within the boundaries of ovnk controlled traffic, meaning it will work in every situation except `hostPort`/`hostNetwork` enabled pods. The same goes for firewalld.
Firewalld rules run before any `NetworkPolicy` is enforced, too. You can have both, they are not incompatible.